### PR TITLE
Home 6

### PR DESCRIPTION
**Pull request by IgorPrihodko**

### CODE REVIEW

File: src/main/java/controller/InitServlet.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import model.User;
+import org.apache.log4j.Logger;
+import service.product.ProductService;
+import service.user.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+
+@WebServlet(value = "/admin/*", loadOnStartup = 0)
```
Comment (Sofasmile) [inline src/main/java/controller/InitServlet.java]: better use: value = "/"

File: src/main/java/controller/user/AllUsersServlet.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@WebServlet(value = "/admin/users", loadOnStartup = 2)
```
Comment (Sofasmile) [inline src/main/java/controller/user/AllUsersServlet.java]: do we need loadOnStartup = 2 here? if yes, write why

File: src/main/webapp/deleteProduct.jsp [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```
Comment (Sofasmile) [inline src/main/webapp/deleteProduct.jsp]: do we need this page for deleting products the same about users?

**Review by Sofasmile — COMMENTED** [review]
resolve some comments from previous review

File: src/main/java/controller/InitServlet.java (hunk 1 above)
Comment (IgorPrihodko) [reply src/main/java/controller/InitServlet.java]: done

File: src/main/java/controller/user/AllUsersServlet.java (hunk 2 above)
Comment (IgorPrihodko) [reply src/main/java/controller/user/AllUsersServlet.java]: fixed

File: src/main/webapp/deleteProduct.jsp (hunk 3 above)
Comment (IgorPrihodko) [reply src/main/webapp/deleteProduct.jsp]: fixed

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**